core/src/main/java/com/novoda/noplayer/SubtitleView.java

```diff
@@ -31,12 +30,12 @@ public SubtitleView(Context context, AttributeSet attrs) {
     }
 
     public void setCues(TextCues textCues) {
```

Comment:
💯  We've updated the mapper, that passes to this, to ensure that it always returns a list.